### DISCUSSION TITLE
rubikck costume graphics issue

### DISCUSSION THREAD

**Discussion by immortalblood** [ValveSoftware/Dota2-Gameplay#9091] [Q&A]
![Screenshot (56)](https://github.com/ValveSoftware/Dota2-Gameplay/assets/133003238/274b774e-f90f-440e-bea6-dabf5a8fa361)

**Comment by AdrianSterling** [comment]
Hi, I'm a newbie. May I ask you what is the name of this game and where can I download it to play? I'm a big fan of high techloky games like this so I'd love to hear back from you.
